### PR TITLE
Add key features

### LINKED ISSUE #426: Add key features
_This issue is part of the migration to v3. You can read full details about the various changes [here](https://github.com/exercism/v3-launch/)._

In Exercism v3, each track must specify exactly six "key features". Exercism uses these features to highlight the most interesting, unique or "best" features of a language to a student.

Key features are specified in the top-level `"key_features"` field in the track's `config.json` file and are defined as an array of objects, as specified in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Goal

The `"key_features"` field in the `config.json` file should be updated to describe the six "key features" of this track. See [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Example

```json
{
  "key_features": [
    {
      "icon": "features-oop",
      "title": "Modern",
      "content": "C# is a modern, fast-evolving language."
    },
    {
      "icon": "features-strongly-typed",
      "title": "Cross-platform",
      "content": "C# runs on almost any platform and chipset."
    },
    {
      "icon": "features-functional",
      "title": "Multi-paradigm",
      "content": "C# is primarily an object-oriented language, but also has lots of functional features."
    },
    {
      "icon": "features-lazy",
      "title": "General purpose",
      "content": "C# can be used for a wide variety of workloads, like websites, console applications, and even games."
    },
    {
      "icon": "features-declarative",
      "title": "Tooling",
      "content": "C# has excellent tooling, with linting and advanced refactoring options built-in."
    },
    {
      "icon": "features-generic",
      "title": "Documentation",
      "content": "Documentation is excellent and exhaustive, making it easy to get started with C#."
    }
  ]
}
```

## Tracking

https://github.com/exercism/v3-launch/issues/5

### PR DESCRIPTION
**Pull request by 2Grey**
Add some key features from [Apple Swift](https://developer.apple.com/swift/)

Closes #426

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1239,7 +1239,38 @@
       "name": "while loops"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Modern",
+      "content": "Swift - result of the latest research on programming languages, combined with of Apple experience.",
```
Comment (ErikSchierboom) [inline config.json:1245]: I made some small tweaks to fix "combined with of" (which is missing some words) and to have the result fit in 100 characters.

```suggestion
      "content": "Swift incorporates the latest research on programming languages, combined with Apple's experience.",
```

File: config.json [hunk 2]
```diff
@@ -1239,7 +1239,38 @@
...
+      "icon": "easy"
+    },
+    {
+      "title": "Cross platform",
+      "content": "Support Apple platforms and Linux, with community members working to port to even more platforms.",
+      "icon": "cross-platform"
+    },
+    {
+      "title": "Open source",
+      "content": "Swift is developed in the open at Swift.org, with source code.",
```
Comment (ErikSchierboom) [inline config.json:1270]:
```suggestion
      "content": "Swift is open-source and developed in the open at Swift.org.",
```

**Review by ErikSchierboom — COMMENTED** [review]
Some minor suggestions

**Review by ErikSchierboom — APPROVED** [review]
Excellent!

**Comment by ErikSchierboom** [comment]
@2Grey The key features can now be seen at https://exercism.org/tracks/swift

**Comment by 2Grey** [comment]
https://exercism.org/tracks/swift/about

**Comment by ErikSchierboom** [comment]
Depends on whether you have joined the track :) The primary goal for key features is to entice new students, so the link I posted is likely the one where the key features are most often viewed.

**Pull request merged**